hw/ip/dma/data/dma.hjson

```diff
@@ -92,6 +95,46 @@
           ]
         }
       ]
+    },
```

Comment:
{ name:     "MODE",
      desc:     '''Set the operational mode of the DMA''',
      swaccess: "rw",
      hwaccess: "hro",
      fields: [
        { bits: "0", name: "MODE",
          desc: "DMA operation mode",
          enum: [
            { value: "0", name: "LINEAR_MODE", desc: "Transfers data linearly"},
            { value: "1", name: "CIRCULAR_MODE", desc: "Transfers data in circular mode"},
          ]
        }
      ]
    },